src/dll/edbgEcmdDll.C

```diff
@@ -1025,9 +1025,17 @@ uint32_t queryConfigExistChips(const ecmdChipTarget &i_target,
       chipData.chipUnitData.clear();
 
       if (isOdysseyChip(ocmbTarget)) {
+        // We should not enter here, as it is odyssey chip
```

Comment:
line 962 to 974 probes pib target but we need to check if it is odyssey and if odyssey we need to call pdbg_target_probe(ocmb) target. This method probes, ocmb, fifo, fsi, pib is used in kernel mode. After that probe we can check if ocmb target is enabled. Noticed we are probing ocmb target at line 1016 so 962 to 974 can be added only to ddr4 ocmb chips

Reply:
Ln: 942 pdbg_for_each_class_target("proc", chipTarget) {
and 
Ln:952 if ((i_target.chipType != ECMD_CHIPT_PROCESSOR) &&

ensures that it is only proc (pu) targets

Ln:1016 pdbg_target_probe(ocmbTarget); takes care of probing ocmb target, which will probe its backend targets.